### PR TITLE
Refs 328: change dev pgsql container name & port

### PR DESCRIPTION
**Pull request by jlsherrill**
this will allow us to use the existing docker compose for spinning up a pulp server

### CODE REVIEW

**Comment by jlsherrill** [comment]
NOTE, this requires a change in all our configs: 

```
database:
  host: localhost
  port: 5432
```
to
```
database:
  host: localhost
  port: 5433
```

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-328

File: mk/db.mk [hunk 1]
```diff
@@ -49,20 +49,20 @@ db-migrate-seed: $(GO_OUTPUT)/dbmigrate ## Run dbmigrate seed
...
 
 .PHONY: db-clean
 db-clean: db-down ## Clean database volume
-	! $(DOCKER) volume inspect postgres &> /dev/null || $(DOCKER) volume rm postgres
+	! $(DOCKER) volume inspect $(DB_CONTAINER_NAME) &> /dev/null || $(DOCKER) volume rm $(DB_CONTAINER_NAME)
 
 .PHONY: db-cli-connect
 db-cli-connect: ## Open a postgres cli in the container (it requires db-up)
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres psql "sslmode=disable dbname=$(DATABASE_NAME) user=$(DATABASE_USER) host=$(DATABASE_HOST) port=$(DATABASE_PORT) password=$(DATABASE_PASSWORD)"
+	! $(DOCKER) container inspect $(DB_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DB_CONTAINER_NAME) psql "sslmode=disable dbname=$(DATABASE_NAME) user=$(DATABASE_USER) host=$(DATABASE_HOST) port=$(DATABASE_PORT) password=$(DATABASE_PASSWORD)"
```
Comment (rverdile) [inline mk/db.mk:60]: it looks like `db-cli-connect` and `db-dump-table` are using a default DATABASE_PORT variable, that is defined `varibles.mk`

Comment (rverdile) [reply mk/db.mk:60]: in this case that values is 5432, so it's causing an error

Comment (jlsherrill) [reply mk/db.mk]: good catch, i split that variable into internal and external port and used them appropriately.

File: mk/db.mk [hunk 2]
```diff
@@ -49,20 +48,20 @@ db-migrate-seed: $(GO_OUTPUT)/dbmigrate ## Run dbmigrate seed
...
 
 .PHONY: db-dump-table
 db-dump-table:
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres pg_dump --table "$(DATABASE_TABLE)" --schema-only --dbname=$(DATABASE_NAME) --host=$(DATABASE_HOST) --port=$(DATABASE_PORT) --username=$(DATABASE_USER)
+	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DATABASE_CONTAINER_NAME) pg_dump --table "$(DATABASE_TABLE)" --schema-only --dbname=$(DATABASE_NAME) --host=$(DATABASE_HOST) --port=$(DATABASE_INTERNAL_PORT) --username=$(DATABASE_USER)
 
 .PHONY: db-shell
 db-shell:
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres bash
+	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(_CONTAINER_NAME) bash
```
Comment (rverdile) [inline mk/db.mk:67]:
```suggestion
	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DATABASE_CONTAINER_NAME) bash
```
Just fix this one thing and it's good to go

**Review by rverdile — APPROVED** [review]
lgtm

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-540

**Pull request merged**